docs/spec.md

```diff
@@ -2084,8 +2097,20 @@ If `batch_group_count > 1`:
     * `padded_input_shape[lhs_dim] = padding[spatial_dim, 0] + dilated_input_shape[lhs_dim] + padding[spatial_dim, 1]`.
     * `dilated_window_shape[lhs_dim] = dim(rhs, rhs_dim) == 0 ? 0 : (dim(rhs, rhs_dim) - 1) * rhs_dilation[spatial_dim] + 1`.
     * `num_windows = (padded_input_shape[lhs_dim] == 0 || dilated_window_shape[lhs_dim] > padded_input_shape[lhs_dim]) ? 0 : floor((padded_input_shape[lhs_dim] - dilated_window_shape[lhs_dim]) / window_strides[spatial_dim]) + 1`.
-* (C27) element_type(`result`) $=$ element_type(`lhs`).
-* (C28) rank(`result`) $= N$.
+* (C26) rank(`result`) $= N$.
+* If the operation uses non-quantized tensors:
+  * (C27 `is_non_quantized_tensor(lhs) and is_non_quantized_tensor(rhs) and
+    is_non_quantized_tensor(result)`.
+  * (C28) element_type(`lhs`) $=$ element_type(`rhs`) $=$ element_type(`result`).
+* If the operation uses quantized tensors:
+  * (C29) `is_quantized_tensor(lhs) and is_quantized_tensor(rhs) and
+    is_quantized_tensor(result)`.
+  * (C30) `storage_type(lhs) =  storage_type(rhs)`.
+  * (C31) `expressed_type(lhs) = expressed_type(rhs) = expressed_type(result)`.
```

Comment:
I think hybrid kernel and mixed precision is a different topic, like in the dot general case?

Reply:
Yes, I also think so. "hybrid kernel" enables different interpretation of the kernel. The essence can  of hybrid kernel is discussed in some of the comment like [here](https://github.com/openxla/stablehlo/pull/1413#discussion_r1183127043). Mixed precision related to allowed types like bf13 and f32 to coexist.

Reply:
Relevant thread: https://github.com/openxla/stablehlo/pull/1413#discussion_r1186530590

Reply:
+1. We got to get to the bottom of this, although this doesn't have to be in this PR. We already have #369 to decide on mixed precision, so I raised its priority since it has repeatedly come up lately.

Reply:
acknowledged.